### ISSUE TITLE
[REQ] Restyle R code

### ISSUE THREAD

**Issue by rcannood** [OpenAPITools/openapi-generator#20080]
Hi @wing328 !

I'm looking into submitting an R package ([`laminr.api`](https://github.com/data-intuitive/laminr.api)) generated by openapi-generator to CRAN.

There a few things that I think CRAN will comment on when I submit the package.

I'd like to make a couple of changes to the mustache templates for R to fix a couple of issues:

* Small code style changes (e.g. using `<-` instead of `=` or `seq_along(x)` instead of `1:length(x)`)
* Fix some issues with the roxygen docs (I get warnings like `default_api.R:2961: @param requires two parts: an argument name and a description.` when running `devtools::document()` so there are some minor things that need to be fixed)
* Fix discrepancies between the httr and httr2 API client -- I see some slight differences in the way that the httr and httr2 versions are being rendered; would it make sense to align their mustache templates a little bit

I'll make sure to not modify the interface of the package as not to break anything for existing packages.

@wing328 : Is it okay if I send a couple of PRs your way? Would you prefer if send one big PR which you can evaluate all at once, or that I try to split up the work into multiple PRs which can reviewed separately?

Also @wing328: do you know of any other R packages that are already on CRAN that were generated by this generator?

Thanks for your input!
Robrecht

**Comment by wing328** [comment]
> @wing328 : Is it okay if I send a couple of PRs your way? Would you prefer if send one big PR which you can review the changes all at once, or that I try to split up the work into multiple PRs which can reviewed separately?

yes please submit 3 separate PRs and we will review accordingly.

>  do you know of any other R packages that are already on CRAN that were generated by this generator?

I do not know. Maybe. If I remember correctly, some users simply upload the R client to github and install directly from there.
